README.md

```diff
@@ -19,7 +19,8 @@ To install `vega` and its dependencies from the Python Package Index using
 ```sh
 pip install jupyter pandas vega
 pip install --upgrade notebook  # need jupyter_client >= 4.2 for sys-prefix below
-jupyter nbextension install --sys-prefix --py vega  # not needed in notebook >= 5.3
```

Comment:
Why is this change needed?

Reply:
Because the line is still necessary in notebook >= 5.3 actually.